TLM/TLM/Custom/PathFinding/CustomPathManager.cs

```diff
@@ -94,14 +99,16 @@ public class CustomPathManager : PathManager {
 
             PathFind[] stockPathFinds = GetComponents<PathFind>();
             int numOfStockPathFinds = stockPathFinds.Length;
-            int numCustomPathFinds = numOfStockPathFinds;
+            // utilize more threads if possible (systems with 8+ threads (9-n))
+            int numCustomPathFinds = Mathf.Clamp(SystemInfo.processorCount / 2, 1, 6);
```

Comment:
You think we need more? What about simulation stuttering because of 6 threads trying to access `PathUnits` on sim thread?

Reply:
Why should sim thread stutter because 6 threads try to access pathunits?
If those 6 threads are run by dedicated cpu threads, there is no task switching involved.
in the worst case 5 of those will hang around and do nothing. 
this would reduce total system throughput due to the os having to manage them.
But I assume we don't care about total system throughput since we want CSL to perform best, not chrome on the second screen or something.
At least that is my current mental model. But I am not an expert in parallel programming, so I could be very wrong here.